[Pages]: Add Framework Guide for Gridsome (#6428)
cloudflare/cloudflare-docs: content/pages/framework-guides/deploy-a-gridsome-site.md

---
pcx_content_type: how-to
title: Deploy a Gridsome site
---

# Deploy a Gridsome site

> [Gridsome](https://gridsome.org) is a Vue.js powered Jamstack framework for building static generated websites & apps that are fast by default. In this guide, you will create a new Gridsome project and deploy it using Cloudflare Pages. You will use the [`@gridsome/cli`](https://github.com/gridsome/gridsome/tree/master/packages/cli), a command line tool for creating new Gridsome projects.

Reviewer: ```suggestion
[Gridsome](https://gridsome.org) is a Vue.js powered Jamstack framework for building static generated websites and applications that are fast by default. In this guide, you will create a new Gridsome project and deploy it using Cloudflare Pages. You will use the [`@gridsome/cli`](https://github.com/gridsome/gridsome/tree/master/packages/cli), a command line tool for creating new Gridsome projects.
```

> ## Installing Gridsome

Reviewer: ```suggestion
## Install Gridsome
```

Install the `@gridsome/cli` by running the following command in your terminal:

```sh
$ npm install --global @gridsome/cli 
```

## Set up a new project

> With Gridsome installed, you can set up a new project by running `gridsome create`. The `create` command accepts a name that defines the directory of the project created and an optional starter kit name. You can find out more in the [Gridsome starters section](https://gridsome.org/docs/starters/).

Reviewer: ```suggestion
With Gridsome installed, set up a new project by running `gridsome create`. The `create` command accepts a name that defines the directory of the project created and an optional starter kit name. You can review more starters in the [Gridsome starters section](https://gridsome.org/docs/starters/).
```
